File: Bungii.Test.Integration/src/test/java/com/bungii/web/pages/driver/Driver_TermsPage.java
package com.bungii.web.pages.driver;

import com.bungii.common.core.PageBase;
import org.openqa.selenium.WebElement;

public class Driver_TermsPage extends PageBase {

    //Terms & Conditions - Text
    public WebElement Text_Terms() { return findElement(".//*[@id='divStep5']/div[3]/div/div/div/table/tbody/tr[9]/td[2]", LocatorType.XPath); }

    //Terms & Conditions - Error - Agree unchecked
    public WebElement Err_Terms () { return findElement("summary5", LocatorType.Id); }

    //Terms & Conditions - H5
    public WebElement Terms_H5 () { return findElement("//div[@id='divStep5']/div[2]/div/h5", LocatorType.XPath); }

    //Terms & Conditions - Agree checkbox
    public WebElement CheckBox_Agree_Click() { return findElement("//input[@id='TermsnConditions.HasAgreedToTerms']/following-sibling::label", LocatorType.XPath); }

    //Terms & Conditions - Agree checkbox
    public WebElement CheckBox_Agree() { return findElement("//input[@id='TermsnConditions.HasAgreedToTerms']", LocatorType.XPath); }

    //Terms & Conditions - Next Button
    public WebElement Button_TermsNext () { return findElement("btnTerms", LocatorType.Id); }
}

File: BungiiAutomation/Bungii.Test.Regression.Web.Integration/Pages/Driver/Driver_TermsPage.cs
using OpenQA.Selenium;
using OpenQA.Selenium.Support.PageObjects;

namespace Bungii.Android.Regression.Test.Integration.Pages.Driver
{
    public class Driver_TermsPage
    {
        public Driver_TermsPage(IWebDriver webdriver)
        {
            PageFactory.InitElements(webdriver, this);
        }

        //Terms & Conditions - Text
        [FindsBy(How = How.XPath, Using = ".//*[@id='divStep5']/div[3]/div/div/div/table/tbody/tr[9]/td[2]")]
        public IWebElement Text_Terms { get; set; }

        //Terms & Conditions - Error - Agree unchecked
        [FindsBy(How = How.Id, Using = "summary5")]
        public IWebElement Err_Terms { get; set; }

        //Terms & Conditions - H5
        [FindsBy(How = How.XPath, Using = "//div[@id='divStep5']/div[2]/div/h5")]
        public IWebElement Terms_H5 { get; set; }

        //Terms & Conditions - Agree checkbox
        [FindsBy(How = How.XPath, Using = "//input[@id='TermsnConditions.HasAgreedToTerms']/following-sibling::label")]
        public IWebElement CheckBox_Agree_Click { get; set; }

        //Terms & Conditions - Agree checkbox
        [FindsBy(How = How.XPath, Using = "//input[@id='TermsnConditions.HasAgreedToTerms']")]
        public IWebElement CheckBox_Agree { get; set; }

        //Terms & Conditions - Next Button
        [FindsBy(How = How.Id, Using = "btnTerms")]
        public IWebElement Button_TermsNext { get; set; }
    }
}
